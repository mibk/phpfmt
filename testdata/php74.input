<?php // PHP 70400


// No trailing comma.
callFn  (
	'arg1',
	arg2
)   ;

// Different precedence of operators.
echo 'Sum: ' . $a + $b;
echo 'Shift: ' . $a << 2;
