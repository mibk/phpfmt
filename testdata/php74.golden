<?php // PHP 70400

#[ this \is just a regular comment ]

// No trailing comma.
callFn(
	'arg1',
	arg2
);

// Different precedence of operators.
echo 'Sum: ' . $a + $b;
echo 'Shift: '.$a << 2;
